build-support/bin/publish_docs.sh

```diff
@@ -85,7 +86,7 @@ if [[ -z "${local_dir}" ]]; then
   do_open "${REPO_ROOT}/dist/docsite/index.html"
 else
   find "${REPO_ROOT}/dist/docsite" -mindepth 1 -maxdepth 1 \
-    | xargs -I '{}' cp -r '{}' "${local_dir}"
+    -exec -I '{}' cp -r '{}' "${local_dir}" +
```

Comment:
What's the deal with the trailing `+` here?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2038.

Can change this to `print0 | xargs -0` if preferred. Both are cryptic to me and I have little preference.

Reply:
Yeah, I think I'd prefer the print0/xargs -0; if nothing else, it's more googlable :)